app/scripts/views/sign_up.js

```diff
@@ -199,6 +201,10 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
         });
     },
 
+    passwordCheck: function () {
+      passwordcheck(this, this.$el.find('#password'), this.metrics, this.translator);
```

Comment:
So, we dont want metrics on whether the bloom filter was used or not? just if its weak or not? we do not want data about the frequency of bloom filter usage? or should i pass in different messages to the callback so we can log all eventualities?